Sources/RichTextKit/RichTextEditor.swift

```diff
@@ -84,7 +84,9 @@ public struct RichTextEditor: ViewRepresentable {
         return textView
     }
 
-    public func updateUIView(_ view: UIViewType, context: Context) {}
+    public func updateUIView(_ view: UIViewType, context: Context) {
+        textView.attributedString = text.wrappedValue
```

Comment:
Sorry, but it might be the case that macOS version of this method also needs to be updated to keep consistency.